source/slang-glslang/slang-glslang.cpp

```diff
@@ -167,6 +283,11 @@ static int glslang_compileGLSLToSPIRV(glslang_CompileRequest* request)
         spv::SpvBuildLogger logger;
         glslang::GlslangToSpv(*stageIntermediate, spirv, &logger);
 
+        if (request->optimizationLevel > SLANG_OPTIMIZATION_LEVEL_DEFAULT)
```

Comment:
Should this be `>=`? It seems like we've got optimizations set up for that level now, but we aren't running them because of this branch.

Reply:
That's right, I missed this change. Good catch. Will change it.

Reply:
You might find that doing so breaks the tests, so I'd be fine with saving that switch for a different pull request, just so that we can deal with any fallout for tests that need their expected output tweaked.